kolibri/plugins/setup_wizard/assets/src/views/importLODUsers/SelectFacilityForm.vue

```diff
@@ -69,7 +69,10 @@
       },
     },
     $trs: {
-      commaSeparatedPair: '{first}, {second}',
+      commaSeparatedPair: {
+        message: '{first}, {second}',
+        context: 'DO NOT TRANSLATE.',
+      },
```

Comment:
This string's ID and context guidance seem suspect.

[Not all languages use ASCII commas](https://stackoverflow.com/a/39286300) as list separators and even if they did in RTL languages it might need to be reordered.

Reply:
Yeah, there's a difference between "DO NOT TRANSLATE THE WORDS INSIDE {}" and "DO NOT TRANSLATE" full stop. I guess if the nuance of the first is being lost, that's why the second has been opted for?

Reply:
"DO NOT TRANSLATE THE WORDS INSIDE {}" is true for almost every string in crowdin

Reply:
Except for the words inside {} inside {}!